### PR TITLE
Signup: Update website type icons in the signup flow.

### PR DESCRIPTION
**Pull request by taggon**
## How to test

You should be able to see the new website type icons when you open `/start/design-type`(with login) or `/start/design-type-with-store`(without login).

Expected:
screen shot 2017-09-06 at 21 18 56
screen shot 2017-09-06 at 21 19 10

This PR is a part of #14863.

cc @cristelrossignol @markryall @fditrapani

### CODE REVIEW

File: client/signup/steps/design-type-with-store/style.scss [hunk 1]
```diff
@@ -134,6 +134,12 @@
 	@include breakpoint( ">480px" ) {
 		display: block;
 	}
+
+	img {
+		display: block;
+		margin: 8.33333% 0 3.3333% 7%;
```
Comment (fditrapani) [inline client/signup/steps/design-type-with-store/style.scss]: I feel like we could use more padding in the cards. I found that this worked nicely:

`margin: 10% auto; width: 85%;`

image

cc @cristelrossignol

**Comment by cristelrossignol** [comment]
I'm seeing the old ones on both login and logged out

File: client/signup/steps/design-type-with-store/style.scss (hunk 1 above)
Comment (cristelrossignol) [reply client/signup/steps/design-type-with-store/style.scss]: Agreed!

**Comment by fditrapani** [comment]
Update looks good to me. Code looks right but it looks like we have another failed test. Maybe another e2e test?

**Comment by alisterscott** [comment]
Functionality this appears to work well.

Is there a reason the images are different sizes? This means the buttons don't align vertically which is obvious when the lower buttons are partially below the fold like this:

design

Also, when showing the four options (new user) and the screen width is above ~960px the boxes get larger vertically so you're more likely to push buttons below the fold, is this intentional?

@taggon: would you mind uploading images inline in Github instead of pasting links to cloudup, since it's easier to read in Github inline and also the content resides historically in Github with the PR/issue whereas in Cloudup it can be removed and become a dead link

**Comment by alisterscott** [comment]
It would be good to get a design review on this - adding the label

**Comment by taggon** [comment]
@alisterscott Thank you. I've uploaded the screenshots and updated the dimension of the new icons as follows:
screen shot 2017-09-07 at 18 00 45

@fditrapani ^^ the CSS is updated too as per your suggestion. I think it would look better the icons have a litter shorter bottom margins.

**Comment by fditrapani** [comment]
@taggon feel free to adjust the bottom margin as you see fit. It was the left/right padding that needed same space which looks great now.

**Comment by alisterscott** [comment]
This is looking great, thanks for the changes 👍

**Comment by taggon** [comment]
@fditrapani @alisterscott @cristelrossignol Thank you all!

**Pull request merged**